### PR TITLE
build: use latest pacific release of ceph

### PR DESCRIPTION
**Pull request by pkalever**
upgrade to ceph version 16.2.0 pacific (stable)

### CODE REVIEW

**Comment by Madhu-1** [comment]
@pkalever can you please check if this is backward compatible with nautilus+ versions of ceph? https://github.com/ceph/ceph-csi#support-matrix

File: build.env [hunk 1]
```diff
@@ -12,8 +12,8 @@
 CSI_IMAGE_VERSION=canary
 
 # Ceph version to use
-BASE_IMAGE=docker.io/ceph/ceph:v15
-CEPH_VERSION=octopus
+BASE_IMAGE=docker.io/ceph/ceph:v16
+CEPH_VERSION=pacific
```
Comment (nixpanic) [inline build.env]: This also requires a go-ceph version that has support for Pacific. If you want to have the features from rbd-nbd, you do not need to update the `CEPH_VERSION`, as that is only used for go-ceph at the moment.

**Comment by mykaul** [comment]
Any progress? It's important Ceph-CSI will support Pacific as soon as possible. Unsure how the interaction between Rook, Ceph-CSI and Ceph works across multiple versions, but https://github.com/rook/rook/releases/tag/v1.6.0 mentions both Pacific and Ceph-CSI 3.3.0.

**Comment by pkalever** [comment]
I still have this item in my TODO list, I will find some time for this tomorrow, thanks for the reminder.

**Comment by Madhu-1** [comment]
@mergifyio rebase

**Review by nixpanic — DISMISSED** [review]

File: build.env (hunk 1 above)
Comment (Madhu-1) [reply build.env]: @pkalever can we just update the base image, not the CEPH_VERSION, go-ceph is not currently supporting the pacific release https://github.com/ceph/go-ceph#supported-ceph-versions

Comment (pkalever) [reply build.env]: Oops! for some reason I thought this PR is merged.
Apologies for delaying, will make an update right now.

Thanks!

Comment (pkalever) [reply build.env]: @Madhu-1 done now.

**Review by Madhu-1 — APPROVED** [review]

**Comment by phlogistonjohn** [comment]
@pkalever - just a heads up:
> Oops! for some reason I thought this PR is merged.
> Apologies for delaying, will make an update right now.

Even if the PR was merged, it would not have found it's way into a released version of go-ceph yet. That said, aside from the yet unsupported build tag and a few other unfortunate build tag choices in the current release, pacific should "just work".

**Comment by Madhu-1** [comment]
> @pkalever - just a heads up:
> 
> > Oops! for some reason I thought this PR is merged.
> > Apologies for delaying, will make an update right now.
> 
> Even if the PR was merged, it would not have found it's way into a released version of go-ceph yet. That said, aside from the yet unsupported build tag and a few other unfortunate build tag choices in the current release, pacific should "just work".

@phlogistonjohn, Yes the build was passing with pacific tag, just to stick with the go-ceph supported versions not updated to pacific build tag for now. This PR was mostly to get the ceph client packages updated to pacific

**Pull request merged**